CovidWatch/app/src/main/java/org/covidwatch/android/ble/BluetoothManager.kt

```diff
@@ -0,0 +1,181 @@
+package org.covidwatch.android.ble
+
+import android.app.*
+import android.content.ComponentName
+import android.content.Context
+import android.content.Context.BIND_AUTO_CREATE
+import android.content.Intent
+import android.content.ServiceConnection
+import android.os.Build
+import android.os.IBinder
+import android.util.Log
+import androidx.core.app.NotificationCompat
+import androidx.core.content.ContextCompat.getSystemService
+import org.covidwatch.android.MainActivity
+import org.covidwatch.android.R
+import org.covidwatch.android.data.ContactEvent
+import org.covidwatch.android.data.ContactEventDAO
+import org.covidwatch.android.data.CovidWatchDatabase
+import org.covidwatch.libcontactrace.BluetoothService
+import org.covidwatch.libcontactrace.BluetoothService.LocalBinder
+import org.covidwatch.libcontactrace.cen.*
+import org.covidwatch.libcontactrace.toBytes
+import org.covidwatch.libcontactrace.toUUID
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+interface BluetoothManager {
+    fun startAdvertiser(cen: Cen)
+    fun stopAdvertiser()
+
+    fun startService(cen: Cen)
+    fun stopService()
+
+    fun changeAdvertisedValue(cen: Cen)
```

Comment:
FYI, this is almost exactly the interface that the cordova plugin had. However, I believe that the iOS interface has the methods `start`, `stop` that handle both the service and the advertiser. You should go ahead and merge this, but we should think about a unified interface for the future.

Reply:
I tried to copy what CoEpi has done so far so they can use it easier. And it seems it worked out https://github.com/Co-Epi/app-android/pull/68